### ISSUE TITLE
[accessibility] Subscribe section form the input is missing a label.

### ISSUE THREAD

**Issue by nisarhassan12** [iterative/dvc.org#1164]
Info about why it is important can be found here: https://dequeuniversity.com/rules/axe/3.4/label

**Comment by jorgeorpinel** [comment]
I don't understand the title, sorry.

Could you also give some examples or specifically where in dvc.org this is an issue in the description? Thanks

**Comment by jorgeorpinel** [comment]
OK got it from the PR #1165 so never mind.

Feel free to skip opening issues if able to submit a PR directly 🙂
